src/workloads/query/CsiFragmentedInsertsFlat.yml

```diff
@@ -0,0 +1,258 @@
+      connectTimeoutMS: 600_000
+
+GlobalDefaults:
+  MaxPhases: &maxPhases 26
+  Database: &db csiFragmentedInsertsFlat
+
+  # The Loader actor creates collections named "Collection<N>" where N corresponds to the thread's
+  # number. We'll use a single collection, created by a single thread, so it becomes 'Collection0'.
+  Collection: &coll Collection0
+
+  # If modifying any of the these parameters, please review 'CsiFragmentedInsertsNested.yml' to
+  # ensure that the results of these two workloads can still be correlated.
+  DocumentCount: &docCount 1e6
+  SchemaWidth: &schemaWidth 10000
+  ObjectWidth: &objectWidth 10
+  SampleSize: &sampleSize 10
+
+ActorTemplates:
+- TemplateName: InsertFromSample
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "Insert"}}
+    Type: SamplingLoader
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: 1024}}]
+        NopInPhasesUpTo: *maxPhases
+        PhaseConfig:
+          Database: *db
+          Collection: *coll
+
+          # No more than sampleSize*objectWidth paths can be "touched" by a sample.
+          SampleSize: *sampleSize
+
+          # When the batch size is the same as sample size, each sampled document will be used in
+          # the batch exactly once, and this will be repeated for the number of batches. We expect,
+          # that the first batch might be affected by "cold" caches but the subsequent batches
+          # would be fully warmed up.
+          InsertBatchSize: *sampleSize
+
+          # The 'SamplingLoader' actor re-samples on repeat, meaning that it would get a new set of
+          # documents likely with different paths and values. This makes each repeat hit different
+          # parts of the indexes, causing a long warm up tail. The instances of this template must
+          # provide a valid "Repeats" value as using the default of -1 is UB.
+          Repeat: {^Parameter: {Name: "Repeats", Default: -1}}
+
+          # The instances of the template must specify the number of batches to make it clear what
+          # they are testing wrt to cold/warm state. Using the default of -1 is UB.
+          # The stats per batch are available in 'IndividualBulkInsert' measurement.
+          Batches: {^Parameter: {Name: "Batches", Default: -1}}
+
+Actors:
+# This workload is completely linear with one actor per phase. Having phases when nothing happens
+# is mildly wastful, but I'm tired of having to adjust the numbers, so assuming "sparse" numbering
+# scheme that is more stable.
+# Phases:
+# 0 setup (no index), 10 create CSI, 20 replace CSI with wildcard index
+# 1,                  11,            21                                  - quiese
+# 3,                  13,            23                                  - insert tail
+# 4,                  14,            24                                  - insert warm
```

Comment:
Why do you call the first insert phase the "tail"?  I find this confusing.  I would have assumed this was just a "cold" or "warming" phase.

Reply:
The goal here was to observe the warm up behaviour and expecting to see a long tail. The test has been dropped per the discussion in https://mongodb.slack.com/archives/C03UE6J2AQ7/p1670952227018319?thread_ts=1670901746.133279&cid=C03UE6J2AQ7